src/api/.rubocop.yml

```diff
@@ -153,6 +153,10 @@ Style/ZeroLengthPredicate:
 Style/RescueModifier:
   Enabled: true
 
+# This cop checks how the *when*s of a case expression are indented in relation to its `case` or `end` keyword.
```

Comment:
how can it be more readable? If the markdown was used for something of course, but as you say it is just a comment. This is not readable: `*when*s`